library/data/registry_entry_attachments/src/main/java/org/quiltmc/qsl/registry/attachment/impl/RegistryEntryAttachmentImpl.java

```diff
@@ -144,6 +146,18 @@ public Set<TagKey<R>> tagKeySet() {
 		return set;
 	}
 
+	@NotNull
+	@Override
+	public Iterator<Entry<R, V>> iterator() {
+		return RegistryEntryAttachmentImpl.this.registry.stream()
```

Comment:
This could probably be made better by using the `keySet()` and `tagKeySet()` methods, to avoid iterating over _every single registry entry_.

Reply:
I tried to make a way that worked well, but this was the easiest. With how fast java is now, this shouldn't be too much of a performance hit.